portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,8 +42,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
       DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
       PreparedQuery results = datastore.prepare(query);
+      List<Entity> limitedResults = results.asList(FetchOptions.Builder.withLimit(2));
```

Comment:
Can you leave this change for a separate PR? Let's focus on the delete in this PR...